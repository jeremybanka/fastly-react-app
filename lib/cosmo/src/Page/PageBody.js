/* @flow */

import * as React from "react";
import { Box } from "../";

type Props = { children: React.Node };

const PageBody = (props: Props) => (
  <Box marginBottom="lg">{props.children}</Box>
);

export default PageBody;
