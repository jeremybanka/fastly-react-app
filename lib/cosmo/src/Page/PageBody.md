The `<Page.Body />` component should be nested inside of a `<Page />`.
