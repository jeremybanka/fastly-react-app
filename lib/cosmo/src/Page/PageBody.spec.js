/* @flow */

import React from "react";
import { shallow, render } from "enzyme";

import PageBody from "./PageBody";

describe("PageBody test suite", () => {
  it("renders with className .PageBody", () => {
    expect(shallow(<PageBody />).is(".PageBody")).toBe(true);
  });

  it("renders the stored snapshot", () => {
    expect(render(<PageBody />)).toMatchSnapshot();
  });
});
