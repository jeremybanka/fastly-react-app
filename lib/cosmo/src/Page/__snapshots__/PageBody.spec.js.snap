// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageBody test suite renders the stored snapshot 1`] = `
<div
  class="PageBody"
/>
`;
